pdf/pdfdocument.cpp

```diff
@@ -119,9 +125,18 @@ void PDFDocument::setSource(const QString& source)
     }
 }
 
+void PDFDocument::requestUnLock(const QString& password)
+{
+  if (!isLocked())
+      return;
+
+  UnLockDocumentJob* job = new UnLockDocumentJob{ password };
```

Comment:
Well, I've just copied what's done for the other object creation in the file, see for instance on line 104. What do you think ?